.github/workflows/release-please.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+name: Release-Please
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+          release-type: python
```

Comment:
my guess would be that this is mainly for making release-please update the setup.py file, which we don't have. So maybe a more generic type would suffice?

Reply:
It mus be one of these listed here: https://github.com/google-github-actions/release-please-action/#release-types-supported. There is no closer match than `python`

Reply:
you could use simple, I think that's what we use in the API

Reply:
I agree, as we do not have any `setup.py`, `setup.cfg` or `version.py` there is no need to use the[ python strategy](https://github.com/googleapis/release-please/blob/main/src/strategies/python.ts).
The simple will update the `CHANGELOG.md` and create a git tag. This should be sufficient, shouldn't it?

Reply:
done